package.json

```diff
@@ -199,9 +199,7 @@
     "ts-jest": "29.1.2",
     "ts-node": "10.9.1",
     "typedoc": "^0.23.10",
-    "typescript": "~4.7.2",
-    "@nx/next": "17.3.0",
-    "@nx/node": "17.3.0"
```

Comment:
Why did we remove these packages?

Reply:
Actually, they didn't get removed, but Yarn automatically alphabetizes the entries in `package.json` (it reduces merge conflicts), and I think someone previously had manually added the entries to the bottom of the file instead of using the yarn add command to add them, so when I added/removed _other_ packages, it re-sorted them to the correct place in the file, making it appear they were removed by me.